src/ArrangerConduit.sol

```diff
@@ -20,15 +22,12 @@ interface RegistryLike {
 // TODO: Use ERC20Helper
 // TODO: Figure out optimal way to structure natspec
 
-contract ArrangerConduit is IArrangerConduit {
+contract ArrangerConduit is IArrangerConduit, UpgradeableProxied {
```

Comment:
In this case there is no storage in `IArrangerConduit`, but it's better practise to always put the required proxy storage at the beginning of the inheritance chain just to minimize the chance of screw up later.

Reply:
Good call, updated